pkgs/applications/version-management/gittyup/default.nix

```diff
@@ -0,0 +1,111 @@
+{ lib
+, mkDerivation
+, fetchFromGitHub
+, cmark
+, mount
+, http-parser
+, perl
+, cmake
+, qtbase
+, qttools
+, git
+, git-lfs
+, ninja
+, libssh2
+, libgit2
+, pkg-config
+, libsecret
+, libgnome-keyring
+, pcre
+, wrapQtAppsHook
+}:
+mkDerivation rec {
+  pname = "gittyup";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "Murmele";
+    repo = "Gittyup";
+    rev = "gittyup_v${version}";
+    fetchSubmodules = true;
```

Comment:
Over half of the vendored  dependencies we do not want to accidentally link against.

```suggestion
```

Reply:
The build fails, if the submodules are not present. As as far as I can see the built binaries does not (dynamically) link against the vendored dependencies.

Reply:
We definitely do not want to accidentally use anything from the [these vendored dependencies](https://github.com/Murmele/Gittyup/blob/master/.gitmodules#L1-L12). They are all security related especially openssl and if I am not mistaken git at least has nixos specific patches.

I think the only option we have is to symlink our source code to the locations since upstream has no support for compiling against system libraries which is a shame.